### DISCUSSION TITLE
Bullet threading

### DISCUSSION THREAD

**Discussion by zakj** [vslinko/obsidian-outliner#533] [Ideas]
I'd love to have bullet threading in Obsidian, and this plugin seems like the right place for it:

![image](https://github.com/user-attachments/assets/dbef74fd-5c48-4c4b-bb4e-e8c183f2bf4f)

The currently-active bullet item gets a highlighted path to the root of the current outline. I find this much easier to get a mental picture of my place in the hierarchy than indentation guides. Seems like there's some other interest:

* https://www.reddit.com/r/ObsidianMD/comments/vrpbgd/is_there_any_bulletthreading_plugin_for_obsidian/
* https://forum.obsidian.md/t/plugin-for-bullet-threading/37317/8
* https://forum.obsidian.md/t/my-css-snippets-for-outline-inspired-by-logseq-bullet-threading-plugin/48558

@vslinko Does this feature makes sense for this plugin? I might have enough motivation to attempt an implementation, but wanted to make sure a PR would be welcome first. :)  (Also, thanks so much for this Outliner! It's making my Logseq transition a lot smoother.)

**Comment by montemartin** [comment]
You could probably implement this as a CSS Snippet without digging into the plugin.
